### ISSUE TITLE
Doesn't work in groups

### ISSUE THREAD

**Issue by KenHV** [nonoo/yt-dlp-telegram-bot#11]
It works perfectly fine in DMs but it doesn't respond to any messages in whitelisted groups. Nothing shows up in logs. I've tried updating all the dependencies but that doesn't work either.

**Comment by nonoo** [comment]
Enable groups in the BotFather settings for your bot. Also try
enabling/disabling group privacy there.

**Comment by KenHV** [comment]
Groups were already enabled. Tried turning off privacy mode, that didn't work either. I kicked the bot and re-added it (with admin rights) to be sure.

**Comment by pmezhuev** [comment]
@KenHV try using `/dlp <url>`

**Comment by KenHV** [comment]
> @KenHV try using `/dlp <url>`

Hey, I just tried it with your patches and it still doesn't work for me. Tried updating dependencies as well.

**Comment by KenHV** [comment]
I added a debug statement at the top of `handleMsg` function, it seems the function isn't triggered at all in groups, no matter what the user and chat IDs are.

cc: @pmezhuev

**Comment by kegfru** [comment]
> Groups were already enabled. Tried turning off privacy mode, that didn't work either. I kicked the bot and re-added it (with admin rights) to be sure.

The same issue - no updates in channels. 
- Groups are enabled
- Privacy mode is disabled

The bot has access to messages (in admin menu)

**Comment by kegfru** [comment]
I found dispatcher.OnNewChannelMessage() works in my case for both channels and groups. 
dispatcher.OnNewMessage() works only for chats

**Comment by kegfru** [comment]
I created issue for td - https://github.com/gotd/td/issues/1495

**Comment by davtur19** [comment]
Yes, it does not work for supergroups but only for basic groups (telegram has this hidden distinction for groups)
